src/closedWindows/closedCard-controller.js

```diff
@@ -3,7 +3,9 @@
 
     class ClosedCardCtrl {
         openClosedWindow(name) {
-            window.windowService.createMainWindow(name, false);
+            var store = window.storeService.open(name);
+            window.windowService.createMainWindow(name, store.isCompact());
+            store.openWindow();
```

Comment:
Does this not belong in the createMainWindow method? Putting it in a controller is not very nice if anything that creates new window will have to call it (store.openWindow)